src/components/LicenseUseCard.vue

```diff
@@ -144,6 +144,10 @@ export default {
     border-top-right-radius: 0.25rem;
 }
 @media only screen and (max-width: 768px) {
+    .tabs li a{
+        font-weight: 600;
```

Comment:
Font weight is correct as-is, you don't need to change it.

Reply:
Font size should be set to `1.125rem` (which would make it 18px).

Reply:
Please, remove the `font-weight` declaration, and set the font size in `rem` units. Otherwise, if the parent li's font gets set to a different value, the font size here would change, too, because `em` units are relative to the parent's font size.